custom_functions/mark_evidence.py

```diff
@@ -1,46 +1,34 @@
-def mark_evidence(container=None, object_id=None, content_type=None, **kwargs):
+def mark_evidence(container=None, input_object=None, content_type=None, **kwargs):
     """
     Mark an object as Evidence in a container
     
     Args:
         container (CEF type: phantom container id): Container ID or Container Object
-        object_id (CEF type: *): Id of object to be added - artifact id, note id, etc.
+        input_object (CEF type: *): The object to marked as evidence. This could be a vault_id, artifact_id, note_id, or if the previous playbook block is an action then "keyword_argument:results" can be used with the content_type "actionrun".
```

Comment:
(sorry, commented on the json above, but the comment should apply to the python)

Two things here:

"to marked as evidence" should be "to be marked as evidence"
container_id is back in, so please add it to the list